Security/SessionDownloadStrategy.php

```diff
@@ -15,6 +15,7 @@
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
```

Comment:
Sorry, its used hereand correct

https://github.com/ahmetakbn/SonataMediaBundle/blob/01bbb0658c66c4ac24699b44a80c7cd9d0101ed3/Security/SessionDownloadStrategy.php#L85

only the order needs to be fixed